src/project.rs

```diff
@@ -112,7 +112,12 @@ impl Project {
     /// on how paths were resolved by PathsResolver
     /// Abort if virtualenv or lock file does not exist
     pub fn install(&self, post_install_action: PostInstallAction) -> Result<(), Error> {
-        print_info_1("Preparing project for development");
+        let message = if self.settings.production {
+            "Preparing project for production"
+        } else {
+            "Preparing project for development"
+        };
+        print_info_1(message);
```

Comment:
Eh. Nitpick, but I'd just put the print_info_1 directly in the if/else, no point in polluting the outer scope with a `message` variable that's only used once.